extension.yaml

```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/reubenmiller/go-c8y-cli/feat/extensions-manager/tools/schema/extension.json
+version: "v1"
+aliases:
+  - name: listSimulators
+    description: Example alias to list oee simulator device
+    command: |
+      c8y devices list --type "c8y_EventBasedSimulator"
+    shell: true
```

Comment:
is that a leftover from copy&paste ?

Reply:
this is the example file that is created automatically when using go-c8y-cli to create a new extension. I'll remove the unnecessary info from it